app/views/events/show.html.erb

```diff
@@ -0,0 +1,102 @@
+<div class="section">
+  <div class="container">
+    <div class="columns is-centered">
+      <div class="column is-5">
+        <div class="box">
+          <h2 class="title is-2 is-spaced has-text-centered mb-6"><%= @event.name %></h2>
+
+          <div class="columns is-mobile is-centered">
+            <div class="column is-5">
+              <p><small>Hosted by</small></p>
+              <h4 class="subtitle is-4"><%= @event.creator.name %></h4>
+            </div>
+            
+            <%# <div class="column"></div> %>
+            <div class="column">
+              <div class="dropdown is-hoverable">
+                <div class="dropdown-trigger">
+                  <button class="button is-fullwidth" aria-haspopup="true" aria-controls="dropdown-menu3">
+                    <span>Invitees: <%= User.find(@event.enrollments.invited.map { |enr| enr.user_id }).count %></span>
+                  </button>
+                </div>
+                <div class="dropdown-menu" id="dropdown-menu6" role="menu">
+                  <div class="dropdown-content">
+                    <ul>
+                      <% if User.find(@event.enrollments.invited.map { |enr| enr.user_id }).any? %>
+                        <div class="dropdown-item">
+                          <% User.find(@event.enrollments.invited.map { |enr| enr.user_id }).each do |user| %>
+                            <li><p><%= user.name %></p></li>
+                          <% end %>
+                        </div>
+                      <% end %>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="column">
+              <div class="dropdown is-hoverable">
+                <div class="dropdown-trigger">
+                  <button class="button is-fullwidth" aria-haspopup="true" aria-controls="dropdown-menu3">
+                    <span>Attendies: <%= User.find(@event.enrollments.accepted.map { |enr| enr.user_id }).count %></span>
+                  </button>
+                </div>
+                <div class="dropdown-menu" id="dropdown-menu6" role="menu">
+                  <div class="dropdown-content">
+                    <ul>
+                      <% if User.find(@event.enrollments.accepted.map { |enr| enr.user_id }).any? %>
+                        <div class="dropdown-item">
+                          <% User.find(@event.enrollments.accepted.map { |enr| enr.user_id }).each do |user| %>
+                            <li><p><%= user.name %></p></li>
```

Comment:
Here as well, Think about how to refactor this into a helper method. Avoid loops, view logic, directly in views. Having the content coming from a helper method is faster than interpreting the content inside a block in the erb file.
Check [this article about rails mistakes, especially the part about view logic](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make#common-mistake-2-putting-too-much-logic-in-the-view).